src/App.tsx

```diff
@@ -1,8 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of posts</h1>
-);
+import { getPost, getUsers, getComments } from './api';
+import { Posts } from './helper';
+import PostList from './PostList';
+
+const App = () => {
+  const [postBefore, setPostBefore] = useState<Posts[]>([]);
+  const [posts, setPost] = useState<Posts[]>([]);
+  const [isLoading, setLoading] = useState(false);
+  const [isLoaded, setLoaded] = useState(false);
+
+  const isLoad = async () => {
+    setLoading(true);
+    const userFromServer = await getUsers();
+    const postFromServer = await getPost();
+    const commentsFromServer = await getComments();
+
+    const preperedListOfPosts = postFromServer.map(item => ({
+      ...item,
+      user: userFromServer.filter(itemId => (itemId.id === item.userId)),
```

Comment:
```suggestion
      user: userFromServer.find(itemId => (itemId.id === item.userId)),
```
тут же должен быть один пользователь?